webpack.config.common.js

```diff
@@ -0,0 +1,71 @@
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/app.js',
+  output: {
+    path: `${__dirname}/dist`,
+    publicPath: '/',
+    filename: 'bundle.js'
+  },
+  devServer: {
```

Comment:
@Dubby20 this should not be here. It should be in the dev config for Webpack

Reply:
Yeah. true. Let me remove that